src/components/common/helpers/time.ts

```diff
@@ -40,3 +40,11 @@ export const formatDurationDiff = (startedOn: string, finishedOn: string) => {
    const diff : moment.Duration = moment.duration(moment(finishedOn).diff(moment(startedOn)))
    return `${diff.hours() > 0 ? `${diff.hours()}h ` : ''}${diff.minutes() >0 ? `${diff.minutes()}m ` : ''}${diff.seconds()}s` 
 }
+
+export const renderDeployedTime = (_env, isAgroInstalled) => {
```

Comment:
Just pass lastDeployed as complete obj is not required. Also, rename `renderDeployedTime` to `processDeployedTime`